vector/src/main/java/im/vector/app/features/call/VectorCallActivity.kt

```diff
@@ -320,6 +328,11 @@ class VectorCallActivity : VectorBaseActivity<ActivityCallBinding>(), CallContro
             is VectorCallViewEvents.ConnectionTimeout -> {
                 onErrorTimoutConnect(event.turn)
             }
+            is VectorCallViewEvents.ShowDialPad -> {
+                CallDialPadBottomSheet.newInstance(false).apply {
+                    callback = dialPadCallback
+                }.show(supportFragmentManager, FRAGMENT_DIAL_PAD_TAG)
```

Comment:
no need to check if already there using tag?

Reply:
It's not really possible to be in this case so I guess its ok